src/organisms/cards/AddCard/index.js

```diff
@@ -11,7 +11,7 @@ function renderContent( props ) {
   var txt, icon;
 
   if ( beneficiary ) {
-    txt = 'Add Beneficiary';
+    txt = 'Add person to policy';
```

Comment:
outside the scope of this PR but I'm still not sure about this component. I was looking at the designs & if you look at 2.4 in the flow, the text changes when you have already added a category, which means we'd need 2 types of component for each state. Like you mentioned last night, the trade off is having to specify `txt` and `icon` when you go to implement the component, but I think this is preferable to having to add all the conditionals for each possible state.